pages/migration/index.rst

```diff
@@ -12,6 +12,31 @@ Migration
 This chapter describes which migrations are performed when upgrading your Nuts node. Most of them will be automatic, but sometimes
 manual action is required.
 
+0.15
+^^^^
+Starting 0.15 the node will default to vendor CA certificates issued by the Nuts Network Authority instead of self-signing
```

Comment:
Is it still possible to add new self-signed certs? Since we might skip the deployment of the network authority entirely...

Reply:
Yes should still be possible, as long as you self-sign the resulting CSR. Then you would need to extract the private key from the crypto store though, which isn't very straightforward to do (since filenames are base64 encoded). If we decided that we probably aren't going the Network Authority Root CA way, maybe we should add another command to crypto for self-signing a certificate (instead of generating a CSR)?

Reply:
just to keep all options on the table.